examples/influxdb-ami/README.md

```diff
@@ -24,9 +24,8 @@ To build the InfluxDB AMI:
 1. To build an Amazon Linux AMI for InfluxDB Enterprise: `packer build -only=amazon-linux-ami influxdb.json`.
 
 When the build finishes, it will output the IDs of the new AMIs. To see how to deploy this AMI, check out the 
-[influxdb-cluster-simple](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-cluster-simple) and
-[influxdb-multi-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-multi-cluster) 
```

Comment:
Why remove the second link? It should prob go to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/tick-multi-cluster

Reply:
okay. I removed it because of `influxdb-multi-cluster` doesn't exist